swap_meet/electronics.py

```diff
@@ -1,2 +1,16 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+class Electronics(Item):
+
+
+    def __init__(self,condition = 0 ):
+        self.category  = "Electronics" 
+        self.condition = condition
+        #super().__init__(condition=condition)
```

Comment:
looks like you used the parent constructor `super()` here. let's clean it up a bit:

```suggestion
        super().__init__(category="Electronics", condition=condition)
```